core/arch/riscv/riscv.mk

```diff
@@ -56,6 +56,17 @@ ifeq ($(CFG_RISCV_SBI_CONSOLE),y)
 $(call force,CFG_RISCV_SBI,y)
 endif
 
+# Disable unsupported and other arch-specific flags
```

Comment:
This works for now, but it doesn't scale well. By the way, the commit message calls these "ARM-specific" flags.

This highlights a problem with the `CFG_` variables. You can set any such variable, but there is no systematic check to see that the variable is supported or even spelled correctly. Fixing that is obviously out of the scope of this PR.

Reply:
Not sure to get it, but edited the commit description to mention unsupported flags, either because their are specific to other architectures or because not tested / not fully implemented.